mtp_api/apps/mtp_auth/migrations/0017_Remove_UserAdmin_Group_from_Security_if_not_FIU.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.17 on 2021-03-31 15:44
+
+from django.db import migrations
+
+def remove_UserAdmin_group_from_security_if_not_FIU(apps, schema_editor):
+    user_model = apps.get_model('auth', 'user')
+    group_model = apps.get_model('auth', 'group')
+    security_users = user_model.objects.filter(groups__name='Security').exclude(groups__name='FIU').all()
```

Comment:
should we exclude super-admins? i.e. us :)

Reply:
Probably, although being a part of the `UserAdmin` group wouldn't make any difference to our ability to create/edit users in Django admin as we are granted that through the inbuilt superadmin toggle. 

But it's maybe something to think about as a wider point; I added a question about that here https://github.com/ministryofjustice/money-to-prisoners-api/pull/647/files to remind myself to talk about it, but then forgot :)